device/tencent/x86_64-tencent_tcs8400-r0/minigraph.xml

```diff
@@ -0,0 +1,63 @@
+<DeviceMiniGraph xmlns:i="http://www.w3.org/2001/XMLSchema-instance" xmlns="Microsoft.Search.Autopilot.Evolution">
```

Comment:
please remove minigraph, no longer used.

Reply:
ok，I'll update the code

Reply:
The code has been updated